oi/OILibraryImpl.cpp

```diff
@@ -35,6 +35,8 @@ extern "C" {
 
 namespace ObjectIntrospection {
```

Comment:
I drew the line here at the minute because I don't want to change the public header for OIL v1. All this stuff is getting ripped out as part of v2 anyway, and that has the correct namespaces.